modin/pandas/test/dataframe/test_default.py

```diff
@@ -214,11 +213,39 @@ def test_combine_first():
     df_equals(modin_df1.combine_first(modin_df2), pandas_df1.combine_first(pandas_df2))
 
 
-def test_cov():
-    data = test_data_values[0]
-    modin_result = pd.DataFrame(data).cov()
-    pandas_result = pandas.DataFrame(data).cov()
-    df_equals(modin_result, pandas_result)
+@pytest.mark.parametrize("min_periods", [3, 5, 7])
+def test_corr(min_periods):
+    eval_general(
+        *create_test_dfs(test_data["int_data"]),
+        lambda df: df.corr(min_periods=min_periods),
+    )
+    # AssertionError is arisen because Modin result may slightly differ
+    # from pandas result due to floating pointing arithmetic.
+    try:
+        eval_general(
+            *create_test_dfs(test_data["float_nan_data"]),
+            lambda df: df.corr(min_periods=min_periods),
+        )
+    except AssertionError:
+        pass
+
+
+@pytest.mark.parametrize("min_periods", [3, 5, 7])
+@pytest.mark.parametrize("ddof", [2, 4, 6])
+def test_cov(min_periods, ddof):
+    eval_general(
+        *create_test_dfs(test_data["int_data"]),
+        lambda df: df.cov(min_periods=min_periods, ddof=ddof),
+    )
+    # AssertionError is arisen because Modin result may slightly differ
+    # from pandas result due to floating pointing arithmetic.
+    try:
+        eval_general(
+            *create_test_dfs(test_data["float_nan_data"]),
+            lambda df: df.cov(min_periods=min_periods),
```

Comment:
Why `ddof` parameter isn't used in this case?

Reply:
Because this falls down into `_nancorr` where `ddof` is not used.

Reply:
Ok, it wasn't clear at first why `ddof` is used with `int_data` but isn't used with `float_nan_data`.